forage-android/src/main/java/com/joinforage/forage/android/ui/AbstractForageElement.kt

```diff
@@ -0,0 +1,26 @@
+package com.joinforage.forage.android.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.LinearLayout
+
+abstract class AbstractForageElement(
```

Comment:
was this class meant to be public? If not, can we make it internal?

Reply:
Ahhhh right on - yes let's make this internal!

Reply:
UPDATE: the compiler says it's gotta public

![image](https://github.com/teamforage/forage-android-sdk/assets/12377418/0c2e83e8-5a42-4897-9da3-7d53719cd736)

At the least, all of the methods on `ForageSDK` specify that they take either a `ForagePANEditText` or a `ForagePINEditText` so I think it's a lot risk than anybody can misuse the `AbstractForageElement` (e.g. by trying to subclass it for their own purposes.

Reply:
gonna mark as resolved but lmk your thoughts